.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Please include JIRA number in the commit message of the PR. -->
+<!--- This will link your PR to the respective JIRA automatically. -->
+
+Many thanks for submitting your Pull Request ❤️!
+
+Please make sure that your PR meets the following requirements:
+
+- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md)
+- [ ] Pull Request title is properly formatted: `RHOAIENG-XYZ Subject`
```

Comment:
this is not valid for ODH

Reply:
@zdtsw well this is a suggestion i wanted to make and something we have been following previously in edge team, to mention the JIRA numbers in the PR title. This would help us in filter out the PR based on its jira.

Reply:
Could we move this information from subject to the message? It pollutes subjects a lot and takes a lot of space.